sources/packages/backend/apps/workers/src/controllers/assessment/assessment.controller.ts

```diff
@@ -399,11 +399,38 @@ export class AssessmentController {
             assessmentId,
             { alternativeReferenceDate: new Date() },
           );
-        // Updates the calculation start date and get the program year totals in parallel.
-        const [, programYearTotalAwards] = await Promise.all([
+        const getSFASApplicationAwards =
```

Comment:
My expectation was the worker controller would not be affected much other than maybe having the below method renamed.
```ts
this.assessmentSequentialProcessingService.getProgramYearPreviousAwardsTotals
```
Is there a benefit having the changes in this way instead what we originally discussed over the call?